### PR TITLE
Feat/scroll sync

### PR DESCRIPTION
**Pull request by RitaBeloMoreira**
implement scroll sync -- PL example

### CODE REVIEW

File: src/columnGrid/ColumnGrid.tsx [hunk 1]
```diff
@@ -366,8 +365,8 @@ export const ColumnGrid: React.FC<ColumnGridProps> = React.memo(
         columnWidth={getColumnWidth}
         height={100} //Its going to be ignored due to autoHeight
         autoHeight
-        scrollLeft={scrollLeft}
-        onScroll={onScroll}
+        // scrollLeft={scrollLeft}
```
Comment (VeraRibeiro) [inline src/columnGrid/ColumnGrid.tsx:368]: can we remove this commented line or is it necessary?

Comment (RitaBeloMoreira) [reply src/columnGrid/ColumnGrid.tsx:368]: I'll leave it there because I don't know if it's needed for other examples

Comment (RitaBeloMoreira) [reply src/columnGrid/ColumnGrid.tsx:368]: I saw that it was commented on before, so I'll just remove it

Comment (VeraRibeiro) [reply src/columnGrid/ColumnGrid.tsx]: thanks!

**Pull request merged**